### PR TITLE
add description how spec modifies attribute defined by HTML5

### PR DESCRIPTION
**Pull request by johanneswilm**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -36,8 +36,11 @@
   <body>
     <section id='abstract'>
       <p>
-       This specification defines allowed values and expected behaviors for the contenteditable attribute.
-       This specification builds on HTML5.
+        This specification defines allowed values and expected behaviors for the contenteditable attribute.
+        This specification builds on HTML5.
```
Comment (marcoscaceres) [inline index.html]:
```suggestion
        This specification builds on [[HTML]].
```

File: index.html [hunk 2]
```diff
@@ -36,8 +36,11 @@
...
+      </p>
+      <p>
+        This specification defines additional values for the contenteditable attribute which is already defined by HTML5.
```
Comment (marcoscaceres) [inline index.html]:
```suggestion
        This specification defines additional values for the {{ElementContentEditable/contentEditable}} attribute which is already defined by [[HTML]].
```

File: index.html [hunk 3]
```diff
@@ -36,8 +36,11 @@
   <body>
     <section id='abstract'>
       <p>
-       This specification defines allowed values and expected behaviors for the contenteditable attribute.
-       This specification builds on HTML5.
+        This specification defines allowed values and expected behaviors for the contenteditable attribute.
```
Comment (marcoscaceres) [inline index.html]:
```suggestion
        This specification defines allowed values and expected behaviors for the {{ElementContentEditable/contentEditable}} attribute.
```

**Review by marcoscaceres — COMMENTED** [review]
Couple of suggestions... note that we HTML dropped the "5" a long time ago.

**Comment by johanneswilm** [comment]
@marcoscaceres Feel free to merge if you are happy with it.

**Pull request merged**